sdk/python/requirements/py3.11-ci-requirements.txt

```diff
@@ -350,12 +350,21 @@ idna==3.7
 imagesize==1.4.1
     # via sphinx
 importlib-metadata==7.1.0
-    # via dask
+    # via
+    #   build
+    #   dask
+    #   jupyter-client
+    #   jupyter-lsp
+    #   jupyterlab
+    #   jupyterlab-server
+    #   nbconvert
+    #   sphinx
+    #   typeguard
 iniconfig==2.0.0
     # via pytest
 ipykernel==6.29.4
     # via jupyterlab
-ipython==8.24.0
+ipython==8.18.1
```

Comment:
why to use older version

Reply:
🤔 not sure, can you try running the same commands on an updated local repo?
```
make install-protoc-dependencies PYTHON=3.9
make install-python-ci-dependencies PYTHON=3.9
make install-protoc-dependencies PYTHON=3.10
make install-python-ci-dependencies PYTHON=3.10
make install-protoc-dependencies PYTHON=3.11
make install-python-ci-dependencies PYTHON=3.11
```